### PR TITLE
Updated docs links to point to new platform

### PR DESCRIPTION
**Pull request by sofietoft**
As we've migrated the documentation to a new platform I've updated most docs links I could find in the CMS project.

I found a few links to the article about the web.config, which is not relevant to 9+.
Let me know if I should also update those - as that will require some updates to the text with the links as well.

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function startUpDynamicContentController($q, $timeout, $scope, dashboardResource
             {
                 title: "Documentation",
                 description: "Find the answers to your Umbraco questions",
-                url: "https://our.umbraco.com/documentation/?utm_source=core&utm_medium=dashboard&utm_content=text&utm_campaign=documentation/"
+                url: "https://docs.umbraco.com/?utm_source=core&utm_medium=dashboard&utm_content=text&utm_campaign=documentation/"
```
Comment (sofietoft) [inline src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js:51]: Not sure if I updated this link correctly, as it looks like it's linked with some tracking as well?

Comment (nikolajlauridsen) [reply src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js:51]: Just had a look at it, and I have to be honest I'm not sure if that link is actually used.

The content of the dashboard (including the links) is actually not managed by the CMS or core team, it's instead fetched from .com  (`https://dashboard.umbraco.com/`, as a grid no less 😱), so this doesn't seem to change the dashboard. 

But regardless, I think this change is fine to merge, but to update the link in the dashboard I think you need to get a hold of COMA unfortunately 😄

**Comment by nikolajlauridsen** [comment]
Other than the small thing I noticed this looks all good to me

**Comment by nul800sebastiaan** [comment]
FYI: I think it's still used as a fallback when the dashboard.umbraco.com site is down / can't be reached!

**Comment by nikolajlauridsen** [comment]
Ahh good to know 😄, then it still makes to update it 👍

File: src/Umbraco.Web.UI.Client/src/views/dashboard/settings/settingsdashboardintro.html [hunk 2]
```diff
@@ -11,7 +11,7 @@ <h5>
         </h5>
         <ul>
             <li>
-                <localize key="settingsDashboard_bulletPointOne">Read more about working with the items in Settings <a class="btn-link -underline" href="https://our.umbraco.com/documentation/Getting-Started/Backoffice/Sections/" target="_blank" rel="noopener">in the Documentation section</a> of Our Umbraco</localize>
+                <localize key="settingsDashboard_bulletPointOne">Read more about working with the items in Settings <a class="btn-link -underline" href="https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/sections/" target="_blank" rel="noopener">in the Documentation section</a> of Our Umbraco</localize>
```
[the remark this answers is not in the document]
Comment (sofietoft) [reply src/Umbraco.Web.UI.Client/src/views/dashboard/settings/settingsdashboardintro.html:14]: Ah! Good catch. I'll fix 💪

File: src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js (hunk 1 above)
Comment (sofietoft) [reply src/Umbraco.Web.UI.Client/src/views/dashboard/dashboard.tabs.controller.js:51]: Have already booked COMA peeps to talk about this, so we should be covered on this front 💪 
But thanks for highlighting it!

**Review by nikolajlauridsen — APPROVED** [review]
Looks good to me now 👍 I'll get it merged when lights are green 🟢

**Pull request merged**